applications/app_balance.c

```diff
@@ -214,7 +214,10 @@ void app_balance_configure(balance_config *conf, imu_config *conf2) {
 
 	// Variable nose angle adjustment / tiltback (setting is per 1000erpm, convert to per erpm)
 	tiltback_variable = balance_conf.tiltback_variable / 1000;
-	tiltback_variable_max_erpm = fabsf(balance_conf.tiltback_variable_max / tiltback_variable);
+	if (tiltback_variable > 0)
```

Comment:
I don't know the project's coding style standards, but in embedded I would always avoid non-braced `if().

```
if (tiltback_variable > 0) {
	tiltback_variable_max_erpm = fabsf(balance_conf.tiltback_variable_max / tiltback_variable);
} else {
	tiltback_variable_max_erpm = 100000;
}
```

Adds only one line and prevents bad things from happening if someone misses that there's no bracing while adding an extra line to the `else` clause.

Reply:
Fixed